mars/core/entity/chunks.py

```diff
@@ -55,5 +58,11 @@ def _update_key(self):
 class Chunk(Entity):
     _allow_data_type_ = (ChunkData,)
 
+    def __str__(self):
+        return f"{type(self).__name__}({self._data.__str__()})"
+
+    def __repr__(self):
+        return f"{type(self).__name__}({self._data.__repr__()})"
```

Comment:
When `__ str__` is not defined, `__repr__` will be used. Hence no redundant methods needed.